content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -40,28 +40,24 @@ The following prerequisites are needed in order to utilize IPv4/IPv6 dual-stack
    * Kubernetes 1.16 or later
    * Provider support for dual-stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)
    * A network plugin that supports dual-stack (such as Kubenet or Calico)
-   * Kube-proxy running in mode IPVS
 
 ## Enable IPv4/IPv6 dual-stack
 
 To enable IPv4/IPv6 dual-stack, enable the `IPv6DualStack` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) for the relevant components of your cluster, and set dual-stack cluster network assignments:
 
+   * kube-apiserver:
+      * `--feature-gates="IPv6DualStack=true"`
    * kube-controller-manager:
       * `--feature-gates="IPv6DualStack=true"`
-      * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>` eg. `--cluster-cidr=10.244.0.0/16,fc00::/24`
+      * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>` eg. `--cluster-cidr=10.244.0.0/16,fc00::/48`
       * `--service-cluster-ip-range=<IPv4 CIDR>,<IPv6 CIDR>`
```

Comment:
Should there also be an example for the `--service-cluster-ip-range` flag like for `--cluster-cidr`? One thing that wasn't obviously for me when I tried dualstack was that the maximum IPv6 CIDR is `/108` (which should somewhere documented?)

Reply:
Sure - would something like this be helpful `eg. --service-cluster-ip-range=10.0.0.0/16,fd00::/108`?

Reply:
There is some more work happening in this area - https://github.com/kubernetes/kubernetes/pull/90115